### PR TITLE
Update compat for Interpolations

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR:
 - Updates the compat for Interpolations
 - Bumps the patch version for a new release

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 
 [compat]
-Interpolations = "0.13"
+Interpolations = "0.13, 0.14"
 julia = "1.7"
```
Comment (claresinger) [inline Project.toml:13]:
```suggestion
julia = "1.7, 1.8"
```

**Review by claresinger — COMMENTED** [review]
Thanks Charlie! Should we also update to julia 1.8 since other components are using that?

File: Project.toml (hunk 1 above)
Comment (charleskawczynski) [reply Project.toml:13]: Sure, I think that's fine

**Review by claresinger — APPROVED** [review]

**Pull request merged**